test/runtests.jl

```diff
@@ -44,6 +44,10 @@ end
     @test typeof(@inferred(Measurement{Float64}(big"3.14"))) == Measurement{Float64}
     @test typeof(@inferred(Measurement{BigFloat}(-5.43f2))) == Measurement{BigFloat}
     @test x != pi != y
+    @test measurement(missing) === missing
+    @test measurement(missing, .1) === missing
+    @test measurement(missing, missing) === missing
+    @test_throws MethodError measurement(1, missing)
```

Comment:
Out of curiosity, why is this a `MethodError`?  Don't you want to make a missing uncertainty a missing value?

Reply:
This case doesn't seem as clear-cut to me. My reasoning is that just because the uncertainty has gone missing somewhere, doesn't mean the actual value is not known and should be `missing`. A `missing` uncertainty might even be interpreted to mean zero uncertainty by some. I think it is better to throw an error here, than just make assumptions for the user.

Reply:
Yeah, makes sense, thanks for the explanation.